MovieApp/MovieApp/MovieApp/MovieViewController.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  ViewController.swift
+//  MovieApp
+//
+//  Created by Five on 23.03.2022..
+//
+
+import UIKit
+
+class MoveViewController: UIViewController {
+
+    var backgroundColor: UIColor
+    
+    init(backgroundColor: UIColor) {
+            self.backgroundColor = backgroundColor
+            super.init(nibName: nil, bundle: nil)
+    }
+    
+    convenience init() {
+            self.init(backgroundColor: .white)
+    }
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        let img = UIImage(named: "deadpoolMovie2.jpg")
+        let imgView = UIImageView(image: img)
+        imgView.contentMode = .scaleAspectFill
+        imgView.clipsToBounds = true
+        //imgView.heightAnchor()
+        
+        
+        let labelName = UILabel()
+        labelName.text = "Deadpool"
+        labelName.textAlignment = .left
```

Comment:
Use .leading instead of .left and .trailing instead of .right. BTW leading is by default.